demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -39,17 +40,11 @@ const productsFragment = gql`
     fragment ProductsGridFuture on Product {
         id
         title
-        status
-        slug
         description
-        type
         price
-        inStock
-        soldCount
+        type
         availableSince
-        image
         createdAt
-        updatedAt
```

Comment:
Why are these fields removed? Does copying even work.

Reply:
it seems the generator was not run after the last changes of products-config. this fields were removed.

> Does copying even work.

copying does only work if all is setup correctly. this is a known issue and will be addressed in a separate pull-request for a different task.